### PR TITLE
Add Photo Supreme for PostgreSQL v3

### PR DESCRIPTION
**Pull request by liljenstolpe**
Add the multi-user / multi-machine version of Photo Supreme v3
with the PostgreSQL v9 backend.

### CODE REVIEW

File: Casks/photo-supreme-postgresql.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  sha256 'ed1d669e46cf073a4720224c0f08aca70da2e6c6687be1a9d733bda9e45a5304'
+
+  url 'http://www.idimager.com/Trial/PhotoSupremePostgreSQL_V3.pkg'
+  name 'Photo Supreme with PostreSQL'
+  homepage 'http://www.idimager.com/WP/?page_id=20'
+  license :commercial
+
+  pkg 'PhotoSupremePostgreSQL_V3.pkg'
+
+# Actually we need >= 10.6.8, but it doesn't look like you can use anything
```
Comment (vitorgalvao) [inline Casks/photo-supreme-postgresql.rb]: Please remove this comment.

File: Casks/photo-supreme-postgresql.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# other than n.m in the version check
+  depends_on :macos => '>= 10.7'
+
+# We need postgresql >=9, but since the default is now at 9.4, might as well
```
Comment (vitorgalvao) [inline Casks/photo-supreme-postgresql.rb]: And this one.

File: Casks/photo-supreme-postgresql.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# just do that
+  depends_on :formula => 'postgresql'
+
+  uninstall :pkgutil => 'com.idimager.idimagersu'
+
```
Comment (vitorgalvao) [inline Casks/photo-supreme-postgresql.rb]: This empty line must go.

File: Casks/photo-supreme-postgresql.rb (hunk 2 above)
Comment (liljenstolpe) [inline Casks/photo-supreme-postgresql.rb]: done

File: Casks/photo-supreme-postgresql.rb (hunk 3 above)
Comment (liljenstolpe) [inline Casks/photo-supreme-postgresql.rb]: Already done - broke the CI

File: Casks/photo-supreme-postgresql.rb (hunk 1 above)
Comment (liljenstolpe) [inline Casks/photo-supreme-postgresql.rb]: done

**Comment by liljenstolpe** [comment]
Made all changes requested

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in e1a6ec353a8087e63177e4edd8cab018f9308529. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**